python/lsst/afw/image/image.cc

```diff
@@ -224,22 +296,35 @@ PYBIND11_PLUGIN(_image) {
         .value("LOCAL", ImageOrigin::LOCAL)
         .export_values();
 
+    declareImageBase<int>(mod, "I");
     declareImageBase<float>(mod, "F");
     declareImageBase<double>(mod, "D");
-    declareImageBase<int>(mod, "I");
     declareImageBase<std::uint16_t>(mod, "U");
     declareImageBase<std::uint64_t>(mod, "L");
 
+    auto clsImageI = declareImage<int>(mod, "I");
     auto clsImageF = declareImage<float>(mod, "F");
     auto clsImageD = declareImage<double>(mod, "D");
-    declareImage<int>(mod, "I");
-    declareImage<std::uint16_t>(mod, "U");
-    declareImage<std::uint64_t>(mod, "L");
+    auto clsImageU = declareImage<std::uint16_t>(mod, "U");
+    auto clsImageL = declareImage<std::uint64_t>(mod, "L");
```

Comment:
Again, for defining types it really doesn't matter. But for overloads it does and the order might be wrong. But we need to look at this on a case by case basis.